### DISCUSSION TITLE
Windows网易云新版本（2.10.5）可以用吗

### DISCUSSION THREAD

**Discussion by hulinhao233** [1715173329/UnblockNeteaseMusic#910] [Q&A] [answered]
我现在是2.9.5可以正常使用，想问一下新版本（2.10.5）可以用吗

**Comment by limbia** [comment]
不行，2.10.3还可以用

**Comment by wukibaka** [comment]
如果不行的话装证书试试

**Comment by aldev814** [comment]
这里测试的是从2.10.2开始就不能用bilibili音源了，最终支持版本是2.9.10

**Comment by ReviveUnblockNCMInstaller** [comment, marked as answer]
UNM v0.27.0 修复了对 2.10.2 及以上版本的支持

**Comment by aldev814** [comment]
又测试了一下，UNM 0.27.1 网易云2.10.8 bilibili音源仍无法使用。在日志中显示正确抓取到音乐地址但是网易云显示无法播放。

**Reply by 1715173329** [reply]
需要设置 endpoint

**Reply by aldev814** [reply]
了解，加入-e https://music.163.com 后正常使用

**Reply by ttttya** [reply]
> 了解，加入-e https://music.163.com 后正常使用
老哥，这个命令去哪运行啊
